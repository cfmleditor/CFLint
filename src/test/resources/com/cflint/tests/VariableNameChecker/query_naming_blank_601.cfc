<cfquery dbtype="query" name=""></cfquery>