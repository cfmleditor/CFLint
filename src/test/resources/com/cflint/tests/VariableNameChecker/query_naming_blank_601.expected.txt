{
  "version" : "",
  "timestamp" : 1749043134,
  "issues" : [ ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ ],
    "countBySeverity" : [ ]
  }
}